lib/auth/api.go

```diff
@@ -39,7 +39,7 @@ type AccessPoint interface {
 	GetNamespace(name string) (*services.Namespace, error)
 
 	// GetServers returns a list of registered servers
-	GetNodes(namespace string) ([]services.Server, error)
+	GetNodes(namespace string, skipValidation bool) ([]services.Server, error)
```

Comment:
let's make this a functional argument to keep the API consistent

so the signature will look like this:

`GetNodes(namespace, SkipValidation())`